### PR TITLE
Generation student_id, based on the record id in the DB

### PR DESCRIPTION
**Pull request by flying-pi**
Added utility methods for generating student_id (based on the User object or user email)

Refactoring existing code to use new methods.

[Link to Youtrack ticket](https://youtrack.raccoongang.com/issue/SKILLONOMY-50)

### CODE REVIEW

File: common/djangoapps/edeos/utils.py [hunk 1]
```diff
@@ -55,6 +58,23 @@ def is_valid_edeos_field(fields):
             return False
     return True
 
+def get_user_id(user_model):
+    # type: (User) -> str
```
Comment (OPersian) [inline common/djangoapps/edeos/utils.py]: This should probably be removed too
(here and in other places)

File: lms/djangoapps/course_wiki/plugins/markdownedx/__init__.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # Make sure wiki_plugin.py gets run.
-from course_wiki.plugins.markdownedx.wiki_plugin import ExtendMarkdownPlugin
+#TODO (flying_pi) commented for run lms
```
Comment (OPersian) [inline lms/djangoapps/course_wiki/plugins/markdownedx/__init__.py]: shouldn't the import be left as is?

File: common/djangoapps/edeos/utils.py (hunk 1 above)
Comment (flying-pi) [reply common/djangoapps/edeos/utils.py]: Hm it is doc-type notation. It used to improve type inference, so it is can be left.  But as it used only in this place - maybe you right.

File: lms/djangoapps/course_wiki/plugins/markdownedx/__init__.py (hunk 2 above)
Comment (flying-pi) [reply lms/djangoapps/course_wiki/plugins/markdownedx/__init__.py]: Thank you, nice catch. It is for my local deployment, I remove it

**Pull request merged**